frontend/ui-pablo/storybook/stories/organisms/auction_information.stories.tsx

```diff
@@ -7,7 +7,7 @@ const AuctionInformationStories = () => {
   const {auctions} = useStore();
   return (
     <Box>
-      <AuctionInformation stats={auctions.activeLBPStats} auction={auctions.activeLBP} />
+      <AuctionInformation stats={auctions.activeLBPStats} auction={auctions.activeLBP} baseAsset={undefined} quoteAsset={undefined} />
```

Comment:
Same comment as in `/storybook/stories/organisms/auction_histories_table.stories.tsx`

Reply:
updated